packages/remix-dev/compiler/plugins/cssImports.ts

```diff
@@ -37,6 +37,7 @@ export function cssFilePlugin({
       build.onLoad({ filter: /\.css$/ }, async (args) => {
         let { metafile, outputFiles, warnings, errors } = await esbuild.build({
           ...buildOps,
```

Comment:
I'd personally prefer to nix this line and be explicit about everything that goes into `esbuild.build` instead of listing an empty array as an override.